omero/sysadmins/cli/usergroup.rst

```diff
@@ -19,15 +19,15 @@ command::
 
 During the addition of the new user, you will need to specify the first and
 last name of the new user and their username as well as the groups the user
-belongs to. To add John Smith as a member of group 2 identified as jsmith,
+belongs to. To add John Smith as a member of the group named ``test-group`` identified as ``jsmith``,
```

Comment:
This still sounds just like if ``test-group`` was identified as ``jsmith``

I think that you need to shift the ``identified as jsmith`` bit to be just behind ``John Smith``.

Then ``To add John Smith, identified as ``jsmith`` as a member of the group named test-group...`` sound better to me